src/timer.rs

```diff
@@ -24,18 +24,25 @@ struct Job {
 struct Schedule {
     main_thread_writer: Sender<Event>,
     jobs: HashMap<u32, Job>,
+    core_jobs: HashMap<u32, Job>,
```

Comment:
There is no way of clearing these jobs. Every time a user calls `blight:reset()` these timers won't be cleared but a new timer will be added when `tasks.lua` loads.

For Èvent::ResetScript` in `main.rs` a method that clears all the core timers needs to be called as well. Otherwise one might queue up a lot of timers. In my use case this will become a lot of timers.